### ISSUE TITLE
[Bug]: Secondary Color Is Not Saved On Vehicles

### ISSUE THREAD

**Issue by Tessio** [Yimura/YimMenu#1471]
### Describe the bug

Since the last update, when changing the secondary color of the street vehicles or your own vehicles it will not be saved.

### Steps To Reproduce

First way to reproduce the bug:
1. Get a street vehicle
2. Modify the secondary color with YimMenu
3. Go to your garage and save the vehicle
4. Now the secondary color strangely disappears completely

Secondary way to reproduce the bug:
1. Get a personal vehicle out of the garage
2. Modify the secondary color with YimMenu
3. Return your vehicle to the garage
4. Now the secondary color strangely disappears completely

Secondary way to reproduce the bug:
1. Get a personal vehicle out of the garage
2. Modify the secondary color with YimMenu
3. Return your vehicle to the garage
4. Now the secondary color strangely disappears completely

Third way to reproduce the bug
1. Get a vehicle out of your garage or get one off the street
2. Modify the secondary color with YimMenu
3. Go to the mechanic (Naturally to save the tunnel in personal vehicles, after tuning it you have to change or buy some things at the mechanic)
4. Now the secondary color strangely disappears completely

### Expected Behavior

Normally when changing the secondary color it should stay on the vehicle as the primary color, maybe it has something to do with VEHICLE::SET_VEHICLE_COLOURS (?

### Screenshots and/or Logs

![2](https://github.com/YimMenu/YimMenu/assets/50866332/a2d2fd67-610e-44fe-8881-df6910b7b8c0)
![1](https://github.com/YimMenu/YimMenu/assets/50866332/ae927b99-6455-42d9-8ba7-0a14674b0757)

### Additional context

Tested on all 3 platforms: Steam, RGL and Epic Games

### Build

- [X] I've confirmed the issue exists on the latest version of YimMenu

**Comment by Tayipc38** [comment]
Did you really bother testing this on all three platforms?

**Comment by Tessio** [comment]
> Did you really bother testing this on all three platforms?

Yes sir, my obsessive compulsive disorder was crying out for me haha

But it is for the common good!!!

**Comment by Tessio** [comment]
Attached to an example video: [Example Video BUG](https://youtu.be/lESBM5m2q9M)

**Comment by Yimura** [comment]
The YimMenu LS Customs does not work too well to keep modifications sticking on the vehicle. (aka. save them in your garage)

It's best to use the `Start LS Customs` button and use that menu to change mods on vehicles if you want them to stick.
![image](https://github.com/YimMenu/YimMenu/assets/24669514/3d2e9d2b-21b4-446d-8a7b-dc8e41578111)

**Comment by gir489returns** [comment]
This is expected behavior, as the game only uses custom colors for demonstrative purposes, if you want to persist the color, you have the change the values that the game uses, which are the actual primary and secondary colors.

**Comment by YimMenuTheBest** [comment]
> Este es un comportamiento esperado, ya que el juego solo usa colores personalizados con fines demostrativos, si desea conservar el color, debe cambiar los valores que usa el juego, que son los colores primarios y secundarios reales.

The strange thing is that before it worked and saved the secondary color, because the main custom color is saved.

**Comment by YimMenuTheBest** [comment]
It looks like something broke it or something changed after the June update.

**Comment by gir489returns** [comment]
Technically what you're asking anyway counts as recovery and YimMenu doesn't support that anyway.
